examples/_client.rb.example

```diff
@@ -2,10 +2,10 @@ require_relative './../lib/oneview-sdk'
 require 'pry'
 
 @client = OneviewSDK::Client.new(
-  url: 'https://oneview.example.com', # or set ENV['ONEVIEWSDK_URL']
+	url: 'https://oneview.example.com', # or set ENV['ONEVIEWSDK_URL']
   user: 'Administrator',  # or set ENV['ONEVIEWSDK_USER']
   password: 'secret123',  # or set ENV['ONEVIEWSDK_PASSWORD']
-  ssl_enabled: false
+  ssl_enabled: false,
```

Comment:
please remove these changes which are not required.

Reply:
@soodpr Changes Updated